src/import/meson.build

```diff
@@ -0,0 +1,70 @@
+# -*- mode: meson -*-
+
+systemd_importd_sources = files('''
+  importd.c
+'''.split())
```

Comment:
Array again, and the other ones in this file too. There seem to be a bunch of these after this, I'm not going to mention them in this review any more.